tests/test_erosita.py

```diff
@@ -0,0 +1,875 @@
+        # for testing purposes I am changing this attribute so the test files get written to the test_data folder
+        self.filtered._top_level_output_path = 'test_data/'
+        # self.raw_data_path is now test_data/erosita_calpv_raw
+
+        # setting up my fake downloaded data
+        # defining my path for fake downloaded data to go in
+        path = 'test_data/erosita_calpv_raw/temp_download/EFEDS/EFEDS/'
+        # making the directories
+        os.makedirs(os.path.dirname(path), exist_ok=True)
+        # making the files
+        obs_ids = ['300007', '300008', '300009', '300010']
+        for obs in obs_ids:
+            with open(path + obs + '.txt', 'w') as f:
+                f.write('testing')
+        # when downloading calpv data they all come with extra pdfs
+            with open(path + obs + 'eRO' + '.txt', 'w') as f:
+                f.write('testing')
+        
+        self.filtered._directory_formatting()
+
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300007/300007.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300008/300008.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300009/300009.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300010/300010.txt'))
+        self.assertFalse(os.path.exists('test_data/erosita_calpv_raw/temp_download'))
+
+    def test_directory_formatting_files_in_three_folders(self):
+        # writing some files to test the function with
+        # for testing purposes I am changing this attribute so the test files get written to the test_data folder
+        self.filtered._top_level_output_path = 'test_data/'
+        # self.raw_data_path is now test_data/erosita_calpv_raw
+
+        # setting up my fake downloaded data
+        # defining my path for fake downloaded data to go in
+        path = 'test_data/erosita_calpv_raw/temp_download/EFEDS/EFEDS/EFEDS/'
+        # making the directories
+        os.makedirs(os.path.dirname(path), exist_ok=True)
+        # making the files
+        obs_ids = ['300007', '300008', '300009', '300010']
+        for obs in obs_ids:
+            with open(path + obs + '.txt', 'w') as f:
+                f.write('testing')
+        # when downloading calpv data they all come with extra pdfs
+            with open(path + obs + 'eRO' + '.txt', 'w') as f:
+                f.write('testing')
+        
+        self.filtered._directory_formatting()
+
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300007/300007.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300008/300008.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300009/300009.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300010/300010.txt'))
+        self.assertFalse(os.path.exists('test_data/erosita_calpv_raw/temp_download'))
+    
+    def test_get_evlist_path_from_obs(self):
```

Comment:
No longer exists - it has been replaced with a BaseMission method (well it is overwritten in a couple of mission classes) get_evt_list_path method

Reply:
Should I change the name in this branch or merge into main and then create a new branch to test the new updates?

Reply:
Like we said in our meeting, I think you're right and merging first is the better idea